build.gradle

```diff
@@ -465,8 +465,8 @@ subprojects {
             dependency "com.github.stefanbirkner:system-rules:1.16.1"
             dependency "org.assertj:assertj-core:2.6.0"
 
-            dependency "org.mockito:mockito-core:2.23.0"
-            dependency "org.mockito:mockito-inline:3.6.28"
+            dependency "org.mockito:mockito-core:3.8.0"
+            dependency "org.mockito:mockito-inline:3.8.0"
```

Comment:
Is there any reason we need to upgrade these version?

Reply:
In some unit-test scenarios, mock static methods are required, and this feature, mockito-inline, was introduced in higher versions, So upgrade the corresponding version.

Reply:
Ok, if so, we can remove the powermock.